src/components/DashboardWidget/DashboardWidget.vue

```diff
@@ -316,7 +316,7 @@ export default {
 		},
 
 		showMoreLabel() {
-			return t('More {what} …', { what: this.showMoreText })
+			return t('More {dashboardItemType} …', { what: this.showMoreText })
```

Comment:
```suggestion
			return t('More {dashboardItemType} …', { dashboardItemType: this.showMoreText })
```

Reply:
Yes, of course. Good, that we need two reviewers :see_no_evil: